src/hostname/hostnamectl.c

```diff
@@ -107,6 +109,11 @@ static void print_status_info(StatusInfo *i) {
         if (!isempty(i->architecture))
                 printf("      Architecture: %s\n", i->architecture);
 
+        if (!isempty(i->hardware_vendor))
+                printf("    HardwareVendor: %s\n", i->hardware_vendor);
+
+        if (!isempty(i->hardware_model))
+                printf("     HardwareModel: %s\n", i->hardware_model);
```

Comment:
here too, output proper English, i.e. add a space between "Hardware" and "Model"